### ISSUE TITLE
Recovery behaviour when robot rotation is failing

### ISSUE THREAD

**Issue by hawesie** [strands-project/autonomous_patrolling#39]
We have seen cases where the robot tries to rotate in place but fails. However driving forward is still possible, and after this rotation is once again. @cburbridge blames our carpet, so let's hope it's that. Anyway, it would be good for something to detect such situations and get the robot moving again.

**Comment by bfalacerda** [comment]
The pull request above basically solved this issue. Two points I'd like to discuss before closing it:
- I created a new node that subscribes to /odom and /cmd_vel and publishes to a new topic /nav_status if the robot is stuck on the carpet. I made it so one can add other relevant monitor information to the topic easily, if needed. This is in a new package that I added in this repo, though I don't know if it should be moved elsewhere.
- As mentioned in issue #27, when move_base is doing the rotate recovery behaviour it is impossible to cancel its goal. Thus, when the robot gets stuck on the carpet while move_base is on recovery, the move_base state cannot be preempted, and the state machine does not go to the recovery state. Given that this is originated on a move_base bug, I just (locally) disabled the recovery behaviours for move_base altogether to avoid the problem. Should I add this option on the launch files?
